### PR TITLE
1 2 user pref

### PR DESCRIPTION
**Pull request by ngilain**
[OO-FEATURE] In the Back Office, users can now update their account via the new "My account page"

### CODE REVIEW

File: BackofficeBundle/Resources/public/coffee/user/LoadUserInfoRoute.coffee [hunk 1]
```diff
@@ -0,0 +1,33 @@
+((router) ->
+  router.route 'user/edit', 'editUserSelf', ->
+    @initDisplayRouteChanges()
+    pageTitle = $('#user-account').data('title')
+    userUrl = $('#user-account').data('user-url')
+    passwordUrl = $('#user-account').data('password-url')
+    editMyAccount pageTitle, userUrl, passwordUrl
+) window.appRouter
+
+editMyAccount = (pageTitle, userUrl, passwordUrl)->
```
Comment (alavieille) [inline BackofficeBundle/Resources/public/coffee/user/LoadUserInfoRoute.coffee]: can you add method inside the router i think is better tahn in namespace window

File: UserAdminBundle/Controller/Admin/UserController.php [hunk 2]
```diff
@@ -63,9 +62,36 @@ public function newAction(Request $request)
...
+
+    /**
+     * @param Request $request
+     *
+     * @Config\Route("/form", name="open_orchestra_user_admin_user_self_form")
+     * @Config\Method({"GET", "POST"})
+     *
+     * @return Response
+     */
+    public function formSelfAction(Request $request)
```
Comment (alavieille) [inline UserAdminBundle/Controller/Admin/UserController.php]: can you add a security for this method

File: UserAdminBundle/Controller/Admin/UserController.php [hunk 3]
```diff
@@ -90,10 +116,35 @@ public function formAction(Request $request, $userId)
...
+
+    /**
+     * @param Request $request
+     *
+     * @Config\Route("/password/change", name="open_orchestra_user_admin_user_self_change_password")
+     * @Config\Method({"GET", "POST"})
+     *
+     * @return Response
+     */
+    public function selfChangePasswordAction(Request $request)
```
Comment (alavieille) [inline UserAdminBundle/Controller/Admin/UserController.php]: may be check than they are an authentificated user

File: UserAdminBundle/Controller/Admin/UserController.php [hunk 4]
```diff
@@ -90,10 +116,35 @@ public function formAction(Request $request, $userId)
...
+        $user = $this->get('security.token_storage')->getToken()->getUser();
+
+        return $this->renderChangePassword($request, $user);
+    }
+
+    /**
+     * @param Request       $request
+     * @param UserInterface $user
+     */
+    public function renderChangePassword(Request $request, UserInterface $user)
```
Comment (alavieille) [inline UserAdminBundle/Controller/Admin/UserController.php]: protected

**Comment by ngilain** [comment]
@alavieille : update done

**Comment by ngilain** [comment]
@alavieille : update done

**Comment by alavieille** [comment]
:+1:

**Pull request merged**